scripts/DGSPlanner/DGSPlannerGUI.py

```diff
@@ -234,6 +278,16 @@ def updateFigure(self):
                                                            Y=1,
                                                            Angle=str(self.masterDict['S2']),
                                                            RelativeRotation=False)
+            elif instrumentName == 'WAND':
+                for bankNum in range(1,9):
```

Comment:
This is not applying the `s2` value. It would also be easier to do this the same way as HYSPEC by setting the log values and reloading the instrument.

```python
mantid.simpleapi.AddSampleLog(Workspace="__temp_instrument", LogName='HB2C:Mot:s2.RBV',
                                              LogText=str(self.masterDict['S2']), LogType='Number Series')
mantid.simpleapi.AddSampleLog(Workspace="__temp_instrument", LogName='HB2C:Mot:detz.RBV',
                                              LogText=str(self.masterDict['DetZ']), LogType='Number Series')
mantid.simpleapi.LoadInstrument(Workspace="__temp_instrument", RewriteSpectraMap=True,
                                                InstrumentName="WaND")
```